SparkPost/2web2ui
# SA-1756: Add Deletion of Scheduled Reports

### What Changed

- Added delete section in edit scheduled reports
- Added delete action in scheduled reports modal.

### How To Test
-  Go to '/signals/analytics' in hibana.
-  Select a saved report that already has scheduled reports. ( Sample Gmail Report! on staging works well.)
-  Click the `Schedule Report` button.
-  In the modal, click the `...` and select delete and then confirm in the new modal. Verify that the scheduled report has been deleted.
 - Click on the name of another scheduled report. This should redirect you to the edit page.
 - Scroll down to the delete panel and click delete and then confirm. Verify that the scheduled report has been deleted.
